<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>584</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>584</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>584</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QalculaTor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/1024px-GNOME_Calculator_icon_2018.svg.png</normaloff>:/images/1024px-GNOME_Calculator_icon_2018.svg.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TwoButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ThreeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eightButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
font: 18pt &quot;Ubuntu&quot;;

color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
font: 18pt &quot;Ubuntu&quot;;

color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(196, 160, 0);
font: 16pt &quot;Ubuntu&quot;;

color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
font: 18pt &quot;Ubuntu&quot;;

color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
font: 18pt &quot;Ubuntu&quot;;

color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrtButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
font: 18pt &quot;Ubuntu&quot;;

color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(239, 41, 41);
font: 15pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dotButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 101, 164);
font: 16pt &quot;Ubuntu&quot;;
color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusMinusButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(117, 80, 123);
font: 18pt &quot;Ubuntu&quot;;

color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>352</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);</string>
   </property>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionBMI_Calc"/>
    <addaction name="actionMetrics"/>
    <addaction name="actionTemperature"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuExtras"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionBMI_Calc">
   <property name="text">
    <string>BMI Calc</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionMetrics">
   <property name="text">
    <string>Metrics</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionTemperature">
   <property name="text">
    <string>Temperature</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
